Hypnotic-Herons/sprint-3
# Db models 2

## Link to Ticket

closes #20 
## Description of Proposed Changes
- user can now add product to sell

## Steps to Test


```
git fetch --all
git checkout addComputer
code .

```

delete migrations
apply migrations

once in my branch run server and go to http://127.0.0.1:8000/products/
then click sell your stuff

look at interwebs see if works
